src/module/pcsclite.cpp

```
@@ -296,8 +304,16 @@ void PCSCLite::HandlerFunction(void *arg)
 
         if (result != (LONG)SCARD_S_SUCCESS)
```

Clang (14.0.6)

[note] +4, including nesting penalty of 3, nesting level increased to 4

(clang)